app/src/main/java/com/example/cocktails/presentation/fragments/favorites/FavoritesFragment.kt

```diff
@@ -31,10 +32,14 @@ class FavoritesFragment : Fragment(R.layout.fragment_favorites) {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        val intent = requireActivity().intent
+        email = intent.getStringExtra("email").toString()
```

Comment:
Extract this string to a constant so we can remove possibility for typo error when passing data. Use that new constant also in the place where we set data inside intent (login fragment)